eng/Testing.props

```diff
@@ -0,0 +1,22 @@
+<Project>
+  <PropertyGroup Condition="'$(RunQuarantinedTests)' == 'true'">
+    <ArtifactsLogDir>$(ArtifactsDir)log\$(Configuration)\Quarantined\</ArtifactsLogDir>
```

Comment:
Out of curiosity, why do we need them to go into a separate directory

Reply:
We don't, I'm cleaning this out.